scrapers/DEU/rki.js

```diff
@@ -0,0 +1,42 @@
+import * as fetch from '../../lib/fetch.js';
+import * as parse from '../../lib/parse.js';
+
+const scraper = {
+  country: 'DEU',
+  timeseries: false,
+  url: 'https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/RKI_Landkreisdaten/FeatureServer/0/query?where=1%3D1&outFields=*&returnGeometry=false&outSR=4326&f=json',
+  type: 'json',
+  curators: [
+    {
+      name: 'Robert Koch Institute',
+      email: ''
+    }
+  ],
+  sources: [
+    {
+      url: 'https://services7.arcgis.com/mOBPykOjAyBO2ZKk/arcgis/rest/services/RKI_Landkreisdaten/FeatureServer/0/query?where=1%3D1&outFields=*&returnGeometry=false&outSR=4326&f=json',
+      name: 'Robert Koch Institute',
+      description: 'Official numbers published by Robert Koch Institute in Germany'
+    }
+  ],
+  async scraper() {
+    const res = await fetch.json(this.url);
+
+    // Data structure
+    // {'features':[{'attributes':{'cases': INT, 'deaths': INT, 'BL': STRING, 'county': STRING, 'EWZ': INT}},..., {'attributes':{'cases': INT, 'deaths': INT, 'BL': STRING, 'county': STRING, 'EWZ': INT}}]}
+
+    const { features } = res; // a list of dictionaries, one for each county
+
+    return features.map(row => {
```

Comment:
NaBD but can wrap the object in parens to avoid the explicit return

Reply:
Sorry, I don't follow the comment. Disclaimer: I'm not really fluent in .js, but I can hack together stuff with good examples from other scrapers. Please feel free to improve!

Reply:
Oh, just that

() = > ({foo:’foo’})
Is equivalent your your current
() => { return {foo:’foo’}}

Doesn’t matter, just shorter :)